music21/metadata/__init__.py

```diff
@@ -1461,15 +1466,25 @@ def title(self):
     def _isContributorNSKey(nsKey: str) -> bool:
         if not nsKey:
             return False
-        prop: PropertyDescription = properties.NSKEY_TO_PROPERTYDESCRIPTION.get(nsKey, None)
+        prop: PropertyDescription = properties.NSKEY_TO_PROPERTY_DESCRIPTION.get(nsKey, None)
         if prop is None:
             return False
 
         return prop.isContributor
 
+    @staticmethod
+    def _needsArticleNormalization(nsKey) -> bool:
```

Comment:
all new methods need a description and test even if private.  I have no idea what this does or when it would be used or how.  THANKS!

Reply:
Type nsKey as str?

Reply:
Missing "str" is there now.  I'll be doing a separate pass adding docs/doctests to _everything_.